include/madrona/physics_assets.hpp

```diff
@@ -67,7 +67,7 @@ struct MeshBVHBuilder {
     static void * build(
         Span<const imp::SourceMesh> src_meshes,
         StackAlloc &tmp_alloc,
-        MeshBVH *out_bvh,
+        render::MeshBVH *out_bvh,
```

Comment:
Why has MeshBVH been moved to the render namespace? MeshBVH is used outside the renderer for envs that need to deal with collisions / geo queries on triangle meshes.